src/Confluent.Kafka/KafkaException.cs

```diff
@@ -23,12 +23,19 @@ namespace Confluent.Kafka
 {
     public class KafkaException : Exception
     {
-        public KafkaException(Error error, string message)
-            : base(message)
+        private Error _error;
```

Comment:
You also have a member with the same name as the type (this is something which happens very frequently, no problem) - you're just using old skool notation to write it out. Yes, what @ewencp says is how to write it out long hand in the current coding style.